utils/pckgen.d/deb/debian/CAS_NAME-modules.dkms

```diff
@@ -0,0 +1,11 @@
+PACKAGE_NAME="<CAS_NAME>-modules"
```

Comment:
How is <CAS_NAME> replaced with actual name? Will the file name be changed as well ("CAS_NAME-modules.dkms") ?

Reply:
Yep. This is done by `utils/pckgen` for all files in this directory.